playbooks/common/openshift-node/config.yml

```diff
@@ -131,8 +131,8 @@
     # TODO: configure these based on
     # hostvars[groups.oo_first_master.0].openshift.hosted.registry instead of
     # hardcoding
-    openshift_docker_hosted_registry_insecure: "{{ openshift.docker.hosted_registry_insecure | default(True) }}"
-    openshift_docker_hosted_registry_network: "{{ hostvars[groups.oo_first_master.0].openshift.common.portal_net }}"
+    docker_hosted_registry_insecure: "{{ openshift_docker_hosted_registry_insecure | default(True) }}"
+    docker_hosted_registry_network: "{{ hostvars[groups.oo_first_master.0].openshift.common.portal_net }}"
```

Comment:
It seems really painful that we are setting this in so many different places. Granted we have copypasta all over the upgrade playbooks.